### PR TITLE
Removing unused traits and factories

### PR DESCRIPTION
**Pull request by awead**
Work done during a Friday refactoring session.

### CODE REVIEW

File: spec/models/user_mailer_spec.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
     let(:message)      { described_class.stats_email(1.day.ago, DateTime.now) }
     let(:mock_service) { double }
     let(:csv)          { "a,b,c\nd,e,f\n" }
-    let!(:generic_work) { create :work, :with_pdf }
+    let!(:generic_work) { create :work }
```
Comment (carolyncole) [inline spec/models/user_mailer_spec.rb:26]: @awead you are removing with pdf here.  Is there a reason?

File: spec/factories/file_sets.rb [hunk 2]
```diff
@@ -35,13 +35,6 @@
       end
     end
 
-    trait :pdf do
```
Comment (carolyncole) [inline spec/factories/file_sets.rb:38]: This was used until you removed it.

**Review by carolyncole — CHANGES_REQUESTED** [review]

File: spec/models/user_mailer_spec.rb (hunk 1 above)
Comment (awead) [reply spec/models/user_mailer_spec.rb:26]: @cam156 Whether or not the work has a pdf is irrelevant to the test. Omitting it has no effect on the outcome.

File: spec/factories/file_sets.rb (hunk 2 above)
Comment (awead) [reply spec/factories/file_sets.rb:38]: @cam156 If the `:with_pdf` work trait isn't needed, then this trait isn't needed either.

**Review by carolyncole — APPROVED** [review]

**Pull request merged**